MealFridge/Models/Partials/MealPartial.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace MealFridge.Models
+{
+    public partial class Meal
+    {
+        public static List<Meal> CreateMealsFromRecipes(List<Recipe> recipes)
+        {
+            var meals = new List<Meal>();
+            foreach (var recipe in recipes)
+            {
+                var type = recipe.Breakfast.Value ? "Breakfast" : recipe.Lunch.Value ? "Lunch" : "Dinner";
```

Comment:
A meal isn't only a breakfast, lunch or dinner, we also track snacks and desserts